Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileCleaner
```

Comment:
You didn't have to make a non static class here. It's still just a stateless class which can clean up files.
Please make it static again.